app/controllers/application_controller.rb

```diff
@@ -1,2 +1,16 @@
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception
+  include SessionsHelper
+
+  def authenticate_user
+    redirect_to new_session_path, flash: {danger: t('alert.sessions.login is needed')} unless logged_in?
+  end
+
+  def correct_user_with_task
+    if current_user.admin == false
+      if current_user.id != Task.find(params[:id].to_i).user_id
```

Comment:
if文のネストは出来るだけ避けた方が良いです。
1行が長くなるのが嫌な場合は、判定部分をメソッドに切り出しても良いです。
```suggestion
    if current_user.admin == false && task.owner?(current_user)
```

Reply:
実装して満足してしまい全く気にしておりませんでした。。
ご指摘ありがとうございます。修正いたしました。
99329a2